src/Components/Account/Account.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+export class Account extends Component {
```

Comment:
Glad we aren't using default exports. That way we can avoid problems when it comes time to export Redux methods